templates/hooks/sdk_delete_post_request.go.tpl

```diff
@@ -0,0 +1,8 @@
+	if respErr == nil {
+		if foundResource, err := rm.sdkFind(ctx, r); err != ackerr.NotFound {
```

Comment:
If a DELETE operation on ReplicationGroup has returned no error, doesn't this mean that the resource will *always* be in a `deleting` status? Is it necessary to do another sdkFind() operation here?

Further, if a DELETE operation on ReplicationGroup returns no error, is there any chance that the ReplicationGroup can end up *not* being deleted in the backend Elasticache service? If there is no chance of this, is it really necessary to do anything here other than return nil and let the elasticache-controller reconcile the resource and remove its finalizer in `cleanup()` and get rid of the CR on the k8s side?